### PR TITLE
net-misc/anydesk: Remove x11-libs/pangox-compat dependency

### PR DESCRIPTION
**Pull request by PPed72**
Closes: https://bugs.gentoo.org/756613
Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>
Package-Manager: Portage-3.0.9, Repoman-3.0.2

### CODE REVIEW

File: net-misc/anydesk/anydesk-6.0.1-r1.ebuild [hunk 1]
```diff
@@ -58,10 +58,12 @@ QA_PREBUILT="opt/${PN}/*"
 src_install() {
 	local dst="/opt/${PN}"
 
-	dodir ${dst}
 	exeinto ${dst}
 	doexe ${PN}
 
+	# bug 706344
+	patchelf --remove-needed libpangox-1.0.so.0 "${ED}"${dst}/${PN} || die
```
Comment (telans) [inline net-misc/anydesk/anydesk-6.0.1-r1.ebuild:65]:
```suggestion
	patchelf --remove-needed libpangox-1.0.so.0 "${ED}${dst}/${PN}" || die
```

This should all be quoted (along with everything else in the ebuild), right?

Comment (PPed72) [reply net-misc/anydesk/anydesk-6.0.1-r1.ebuild:65]: I just copied that from 6.0.0-r1 ebuild. Will fix it as you indicated.

**Comment by PPed72** [comment]
@telans : fixed quotations. Thanks for the tip.

**Review by a17r — CHANGES_REQUESTED** [review]
Thanks for your fix. Don't be surprised that I dropped the revbump: No one was able to build this package yet because of the inexistent dependency.

**Review by a17r — APPROVED** [review]

**Pull request closed without merging**